src/ArticleQualityService.ts

```diff
@@ -163,7 +163,6 @@ class ArticleQualityService {
     revisions: WikidataResponseParsed
   ): Promise<OresScoresResponse | void> {
     const validRevisions = Object.values(revisions).filter(rev => !rev.missing);
-
```

Comment:
This is just the removal of a line, but I will use this space as a reminder to remove this now unused method and the associated class properties (`this.oresHost`).